dashboards-observability/common/types/explorer.ts

```diff
@@ -110,3 +110,39 @@ export interface IExplorerProps {
   setEndTime?: any;
   appBaseQuery: string;
 }
+
+export interface SavedQuery {
+  description: string;
+  name: string;
+  query: string;
+  selected_date_range: { start: string; end: string; text: string };
+  selected_fields: { text: string; tokens: [{ name: string; type: string }] };
+  selected_timestamp: { name: string; type: string };
+}
+
+export interface SavedVisualization {
+  description: string;
+  name: string;
+  query: string;
+  selected_date_range: { start: string; end: string; text: string };
+  selected_fields: { text: string; tokens: [] };
+  selected_timestamp: { name: string; type: string };
+  type: string;
+  application_id?: string;
```

Comment:
do we need the `application_id` member in this interface? I remember this was an unneeded one?

Reply:
Oh yes, we do need it for saved visualizations since that's the property that associates it with an application. The ones that were unneeded were savedVisualization under savedQuery and savedQuery under savedVisualization.